02-microarray/pathway-analysis_microarray_02_gsea.Rmd

```diff
@@ -394,19 +421,26 @@ head(t_vector)
 
 ### Run GSEA using the `GSEA()` function
 
-Genes were ranked from most highly positive to most highly negative, weighted according to their gene-level statistic, in the previous section.
-In this section, we will implement GSEA to calculate the enrichment score (ES) for each gene set using our pre-ranked gene list.
+Genes were ranked from most positive to most negative, weighted according to their gene-level statistic, in the previous section.
+In this section, we will implement GSEA to calculate the enrichment score for each gene set using our pre-ranked gene list.
+
+The GSEA algorithm utilizes random sampling so we are going to set the seed to make our results reproducible.
+
+```{r}
+# Set the seed so our results are reproducible:
+set.seed(2020)
```

Comment:
> We set the seed in the RNA-seq example, but we didn't here. I added a set.seed() step here, right before we "need" it should someone be viewing this example and have jumped to the analysis code. Is this the correct thing to do?

Yes, this is correct. At the time of authoring this example, it was not obvious that the `set.seed()` step also needed to be added before the `GSEA()` function, which is why it was left out here. Good point re the placement of the step as well!